backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -98,6 +99,20 @@ module.exports = class CrmPage {
     await Promise.all([
       this.engine.click(selector),
       this.engine.waitForNavigation()
-    ]);      
+    ]);
+  }
+
+  /**
+   * Checks if there are no price fields present on the page or not
+   *
+   */
+  async checkIfPriceFieldsAreEmpty () {
+    // checks for the messages div - appears if the list is empty.
```

Comment:
No need for this comment, the code is self explanatory enough.

Reply:
Done!